Support function void argument

splasky opened the pull request:
Void argument is recommend to use inside the function which has
no parameter.This commit can allow user define void during function
declaration.

pfalcon commented on shivyc/tree/nodes.py:
```diff
@@ -668,6 +668,10 @@ def extract_params(self, decl):
 
         for param in func_decl.args:
             decl_info = self.get_decl_infos(param)[0]
+            # check if void is the function argument,we skip
```
The patch (IMHO, as an outsider to the project) looks pretty clean, the only suggestion is with this code comment. Would be nice to reword/reformat it a bit, e.g. "check if "void" is the function argument, and skip it".

pfalcon commented:
@ShivamSarodia: This seems to be more or less good patch, and definitely a needed feature, any response?

splasky committed:
Support function void argument

Void argument is recommend to use inside the function which has
no parameter.This commit can allow user define void during function
declaration.

ShivamSarodia commented:
Hi there! I really appreciate your contribution here. Unfortunately, this project is no longer under active development so I will close out this PR for now.
